environment-mac.yml

```diff
@@ -3,56 +3,54 @@ channels:
   - pytorch
   - conda-forge
 dependencies:
-  - python==3.9.13
-  - pip==22.2.2
+  - python>=3.10, <3.11
+  - pip>=22
```

Comment:
Whoops! Almost missed this (and it's even my fault - I didn't know better at the time) - 'pip' should never be in the dependencies.

Reply:
I am not used to Conda, so I just did what felt right and changed it to use the most recent one 🙈

Reply:
Same! I only understood much later that it shouldn't be done :)